src/components/BottomModalLayout/RestaurantDetailModal/RestaurantDetailModal.jsx

```diff
@@ -0,0 +1,24 @@
+import styles from "./RestaurantDetailModal.module.css";
+
+function RestaurantDetailModal({ restaurant, onClose }) {
```

Comment:
```suggestion
function RestaurantDetailModal({ restaurant: { name, description }, onClose }) {
```
https://github.com/woowacourse/self-paced-react/pull/31#discussion_r2036450505
저는 개인적으로 구조 분해 할당 문법을 많이 사용하는 것 같은데 카멜은 어떠신가요??😮

Reply:
좋네요!! 저는 갠적으로 코드 내에서 ```restaurant.name``` 으로 쓰는 게 레스토랑의 이름이라는 느낌이 직관적이라서 인자에서 구조분해할당을 자주 안쓰는 것 같아요! 혹은 다른 인자로 만약에 똑같이 name 속성이 있는 객체를 받게된다면 이러한 방식의 통일성이 깨질 것 같아서요!

하지만 지금 구조에서는 restaurant 객체 하나만 받고있기 때문에 써밋의 방식이 더 가독성을 높여줄 것 같네요!

Reply:
오 그러네요 동일한 속성을 가진 두 객체가 있다면 일반적인 방식으로 구조 분해 할당 문법을 사용하기 어려울 것 같긴하네요!

근데 생각해보면 그런 상황이 오히려 특수한 경우라는 생각이 들어서(아닌 경우가 더 많을 것 같은 느낌..?) 저는 기본적으로 구조 분해 할당 문법을 사용하고 카멜이 말해준 것처럼 속성이 겹치는 경우에는 아래와 같이 별칭을 붙여서 사용할 것 같아요!🙃
```ts
const people = {
  name: "John",
  age: 29,
};

const student = {
  name: "Jane",
  address: "123 Main St",
};

const { name: peopleName } = people;
const { name: studentName } = student;
```

Reply:
호옹 이렇게 별칭만드는 걸 그럼 써밋처럼 인자에서 구조분해할당을 한다면
```js
function 써밋({
  person: { name: personName },
  student: { name: studentName },
}) {
...
}
```
요렇게만 해주면 되겠네요!👍
좋은데요~~